src/Microsoft.Identity.Web/AuthorizeForScopesAttribute.cs

```diff
@@ -128,9 +124,13 @@ private bool CanBeSolvedByReSignInOfUser(MsalUiRequiredException ex)
                  OidcConstants.ScopeProfile,
             };
 
+            // TODO: scopes can actually be null here - how do we treat this case? 
```

Comment:
This is the kind of bugs that can be discovered. In this particular case I am not sure how to handle it - please advise.

Reply:
yes...good point. This is for incremental consent, so we technically should only be here if we have scopes, because that's the point, that additional scopes are required. So I think in `OnException()`, we should make sure we have something. @jmprieur thoughts?

---
In reply to: [433215274](https://github.com/AzureAD/microsoft-identity-web/pull/183#discussion_r433215274) [](ancestors = 433215274)